SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -61,7 +61,7 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
     appSecretLabel?.stringValue = UserDefaults.standard.string(forKey: kMSAppSecret) ?? appCenter.appSecret()
     logURLLabel?.stringValue = (UserDefaults.standard.object(forKey: kMSLogUrl) ?? prodLogUrl()) as! String
     userIdLabel?.stringValue = showUserId()
-    setEnabledButton?.state = appCenter.isAppCenterEnabled() ? NSControlStateValueOn : NSControlStateValueOff
+    setEnabledButton?.state = appCenter.isAppCenterEnabled() ? NSControl.StateValue.on : NSControl.StateValue.off
```

Comment:
Same as in comment above: `setEnabledButton?.state = appCenter.isAppCenterEnabled() ? .on : .off`?